src/PaymentMethods/PaymentMethodFactory.php

```diff
@@ -120,7 +122,6 @@ class PaymentMethodFactory
         Przelewy24::class => ['przelewy24'],
         PointOfSale::class => ['pospayment'],
         Giropay::class => ['giropay'],
-        NoServiceSpecifiedPayment::class => ['noservice'],
```

Comment:
why is this removed?

Reply:
it was writen twice, you can check line 120 is still there